osu.Game/Screens/SelectV2/SongSelect.cs

```diff
@@ -666,14 +679,17 @@ private void updateWedgeVisibility()
 
         private void updateBackgroundDim() => ApplyToBackground(backgroundModeBeatmap =>
         {
-            backgroundModeBeatmap.BlurAmount.Value = 0;
             backgroundModeBeatmap.Beatmap = Beatmap.Value;
             backgroundModeBeatmap.IgnoreUserSettings.Value = true;
+
+            backgroundModeBeatmap.BlurAmount.Value = 0;
```

Comment:
Does this set have a purpose? I'd like to think it doesn't given this appears to always get overwritten by the one at the end of the method...

Reply:
It does not, removed.